src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.Tests/Azure.ClientSdk.Analyzers.Tests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net472</TargetFrameworks>
```

Comment:
Does this work the way you expect? In other words, does this target end up in the package? Or does that only happen if we build on windows?

Reply:
Oh, that's a good point. This is only the test project so there's no packaging, but if we never run it on Windows it won't matter.

Reply:
You're right: we only build and test on Ubuntu. I'll open a separate track issue.